filesystems/native.py

```diff
@@ -59,14 +59,23 @@ def _remove_file(fs, path):
         raise
 
 
-def _create_directory(fs, path, with_parents):
+def _create_directory(fs, path, with_parents, allow_existing):
     try:
         if with_parents:
-            os.makedirs(str(path))
+            # TODO: it works fine without this conditional passing of
```

Comment:
For a mature library (which Filesystems isn't yet...) I'd go with "we should handle it", but until then I think it's safer to let the Py3 API handle it until we're less likely to have a bug than it is (which yeah not yet I think).

Reply:
Especially since venvs/filesystems can probably safely drop Py2 support at this point.